src/indicators/idmc/update_idmc.R

```diff
@@ -114,7 +114,7 @@ df_idmc_flags_new <- dplyr$anti_join(
   dplyr$group_by(iso3) |>
   dplyr$mutate(
     email = ifelse(
-      Sys.Date() - end_date > 60 | start_date != max(start_date), # do not email for old events updated or if not the latest alert
+      Sys.Date() - end_date > 60 | start_date != max(start_date, -Inf), # do not email for old events updated or if not the latest alert
```

Comment:
whats the addition of  `-Inf` doing here? I'm sure it's fine, just thought easier if you explain

Reply:
Removing some of those warnings that you were seeing in the GitHub actions. Basically if you have a data frame with no rows (sometimes no new flags), then a warning is generated from `max(start_date)` because it's a vector of `0` length. `-inf` just simply removes that warning so that `max()` returns some value at all times.